test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow.sh

```diff
@@ -7,11 +7,11 @@ set -ex
 # Function to install kfctl cli
 install_kfctl(){
 
-    KFCTL_VERSION="v1.0.2"
+    KFCTL_VERSION="v1.2.0"
```

Comment:
is this the latest version?

Reply:
yes. See source [here](https://github.com/kubeflow/kfctl/releases).